Assets/_BForBoss/_Utility/Scripts/Tags.cs

```diff
@@ -1,7 +1,14 @@
+using UnityEngine;
+
 namespace Perigon.Utility
 {
     public static class Tags
     {
         public static string Player => "Player";
     }
+
+    public static class Layers
+    {
+        public static int TriggerArea => 1 << LayerMask.NameToLayer("TriggerArea");
+    }
 }
```

Comment:
Do you think we should rename the script to `TagsAndLayers.cs` or create a new script called `Layers.cs`. Cause I can definitely seeing someone forgetting this inline class existing in the future and creating a new one later on

Reply:
I think I have an idea
TagsAndLayers Class That contains Tags class and Layers class